clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/ops/AbstractEnableDisableAtomicOperation.groovy

```diff
@@ -76,14 +78,29 @@ abstract class AbstractEnableDisableAtomicOperation implements AtomicOperation<V
         return true
       }
 
+      //If LB and TG is present and desiredPercentage is present -> throw error
+      //Deregistering specific tasks from an ALB is currently not supported for Titus
```

Comment:
I think you can just remove the comment here since the if/log already provide the right info. Besides, the comment says it will `throw` but there is no throwing here, expected?

Along with this I would also augment the log statement to something like
```java
log.error(
          "Could not {} server group '{}' in region {}! Disabling by percentage for server groups with target groups not supported by Titus",
          verb, serverGroupName, region
```

Reply:
Makes sense , i'll make the updates